.github/workflows/e2e-stress-test-flake-fix.yml

```diff
@@ -20,6 +20,15 @@ on:
         type: string
         required: false
         default: "ee"
+      qa_db:
+        description: 'Select the type of Database you want to run your tests with'
+        type: choice
+        required: true
+        default: "none"
+        options:
+          - none
+          - postgres
+          - mongo
```

Comment:
I would maybe make this optional and remove "none".
So if you don't provide anything, we just run tests without QA DBs. WDYT?

Reply:
That could work, but mechanically, it's the same as making it required and supplying a default value. Technically you don't need to select anything. I just think of in terms of the UX, having optional dropdown might be weird. Let's say we have 2 choices `sql` and `mongo` once you select either one of the two, you can't unselect it. Providing `none` allow you to chanoge that.

What do you think?

Reply:
OMG!
I totally overlooked that you provided `default: "none"` 😶‍🌫️